XCTestBootstrap/Configuration/FBTestLaunchConfiguration.m

```diff
@@ -27,14 +27,16 @@ - (instancetype)initWithTestBundlePath:(NSString *)testBundlePath applicationLau
   _testHostPath = testHostPath;
   _timeout = timeout;
   _shouldInitializeUITesting = initializeUITesting;
+  _testsToRun = testsToRun;
+  _testsToSkip = testsToSkip;
 
   return self;
 }
 
 + (instancetype)configurationWithTestBundlePath:(NSString *)testBundlePath
 {
   NSParameterAssert(testBundlePath);
-  return [[FBTestLaunchConfiguration alloc] initWithTestBundlePath:testBundlePath applicationLaunchConfiguration:nil testHostPath:nil timeout:0 initializeUITesting:NO];
+  return [[FBTestLaunchConfiguration alloc] initWithTestBundlePath:testBundlePath applicationLaunchConfiguration:nil testHostPath:nil timeout:0 initializeUITesting:NO testsToRun:nil testsToSkip:nil];
```

Comment:
We could do that. But then later in `XCTestConfiguration` an empty set for `testsToRun` means since Xcode 8.3 beta "don't run any tests". So if we define `nonnull` here in `FBTestLaunchConfiguration`, we need to react to that later on when composing the `XCTestConfiguration`. Would you prefer that? I'll change it then, just let me know :).